ArmVirtPkg/ArmVirtKvmTool.dsc

```diff
@@ -90,6 +90,7 @@
 [LibraryClasses.AARCH64]
   ArmCcaLib|ArmVirtPkg/Library/ArmCcaLib/ArmCcaLib.inf
   ArmCcaRsiLib|ArmVirtPkg/Library/ArmCcaRsiLib/ArmCcaRsiLib.inf
+  RngLib|MdePkg/Library/BaseRngLib/BaseRngLib.inf

```

Comment:
In DXE ArmVirt already uses RNDR if available, via DxeRngLib -> EFI_RNG_PROTOCOL -> RngDxe -> BaseRngLib.inf

Using BaseRngLib.inf unconditionally isn't a good idea I think.  There is too much hardware in the wild which has no RNDR support.